index.js

```diff
@@ -1,445 +0,0 @@
-};
-
-/*------------------------Gameplay Functions-------------------------*/
-===
-=== =
-/*-------------------------------Gameplay Functions--------------------------------*/
-//steralizes user inoput to check against command word bank
->>>
->>> > 94 cb5f87c00a9964209b9e74dc9704ffd8edf4fa
-
-function cleanWords(word) {
-  let steralize = word.toString().trim().toLowerCase();
-  return steralize;
-}
-//function that counts down until the impeding zombie horde kills the player
-function zombieHorde() {
-  if (player.encroacment >= 0) {
-    player.encroacment = player.encroacment - 1;
-    if (player.encroacment === 12) {
-      console.log(
-        "You can hear the sound the zombie hoarde growing louder and louder"
-      );
-    }
-    if (player.encroacment <= 6 && player.encroacment > 0) {
-      console.log(
-        "You can see the zombies slowly moving towards you, they are now focused on you!"
-      );
-    }
-    if (player.encroacment === 0) {
-      console.log(
-        "The zombies have gotten you and added you to their ranks. Enjoy the brains!"
-      );
-      process.exit();
-    }
-  }
-}
-
-//uses item to create space from the horde
-function hordeGain() {
-  if (player.inventory.includes(word) && word === 'runningShoes') {
-    encroachment = 20
-    console.log("Your new kicks allow you to put some distance between yourself and the zombies!")
-  } else {
-    console.log("You can't use that")
-  }
-}
-
-<<
-<< << < HEAD
-
-function takeItem(word) { //function for taking items
-  ===
-  === =
-  //picks up takeable items and stashes them in players array
-  function takeItem(word) {
-    >>>
-    >>> > 94 cb5f87c00a9964209b9e74dc9704ffd8edf4fa
-    let pickUpItem = itemLookUp[word]
-    if (pickUpItem.takeable === true && player.location.inv.includes(word)) {
-      player.location.inv.splice(player.location.inv.indexOf(word), 1)
-      player.inventory.push(word)
-      console.log("You picked up " + word)
-    } else {
-      console.log("This is not yours to take!")
-    }
-  }
-
-  <<
-  << << < HEAD
-
-  function unlock(word) { //function for unlock door
-    ===
-    === =
-    //unlocks final room so player can escape to safety
-    function unlock(word) {
-      >>>
-      >>> > 94 cb5f87c00a9964209b9e74dc9704ffd8edf4fa
-      if (player.inventory.includes('key')) {
-        currentState.locked = false
-        console.log("You unlocked the padlock! Now quickly make your way into the tunnel and make your way to Paradise Cove")
-      } else {
-        console.log("You don't have a key to do that")
-
-      }
-    }
-
-    //allows plater to examine item
-    function examine(word) {
-      let lookAt = itemLookUp[word]
-      if (player.location.inv.includes(word)) {
-        console.log(lookAt.desc)
-      }
-    }
-
-
-    /*----------------------------Player-------------------------------*/
-    //player state 
-    const player = {
-      inventory: [],
-      location: null,
-      encroacment: 20,
```

Comment:
There is a typo here that breaks your program on line 409 stopping you from reaching your await ask